.github/actions/create-pr/action.yml

```diff
@@ -64,7 +64,7 @@ runs:
       name: Git client setup and refresh tip
       run: |
         git config user.name "Powertools for AWS Lambda (Python) bot"
-        git config user.email "aws-lambda-powertools-feedback@amazon.com"
+        git config user.email "aws-powertools-maintainers@amazon.com"
```

Comment:
use the noreply email address instead of this one: 
`git config user.email '151832416+aws-powertools-bot@users.noreply.github.com'`

Reply:
great catch! Fixing shortly